docs/languages/polyglot.md

```diff
@@ -0,0 +1,100 @@
+# Polyglot Notebooks
+
+## What is Polyglot Programming?
+
+Polyglot programming is the practice of using multiple programming languages to leverage the strength of each language for different tasks.
+
+## What are Notebooks?
+Notebooks are interactive files that allow the mixing of executable code, visualizations, equations and narrative text. Notebooks are composed of code cells that make it easy to quickly iterate on code. Created and popularized by the open-source project [Jupyter](https://jupyter.org/), they have become the de facto tool for data science and a great resource for teaching or learning a new programming language and quick prototyping.
+
+
+
+## Polyglot Notebooks
+
+When polyglot programming meets notebooks, you get Polyglot Notebooks! As opposed to traditional notebooks that are typically used with Python, the [Polyglot Notebooks Extension](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.dotnet-interactive-vscode) allows you to use multiple programming languages natively all in the same notebook! No more needing
+
+![Polyglot Notebooks](images/polyglot/polyglot_ext.png)
+
+### Languages Supported
+- C#
+- F#
+- PowerShell
+- JavaScript
+- HTML
+- Mermaid
+- SQL
+- KQL (Kusto Query Language)
+
+
+## Features
+In addition to executing the code for the languages above, you can take advantage of:
+
+- **Connecting to and querying from** Microsoft SQL Server databases and Kusto clusters
+
+- **Language server support** such as intellisense, autocompletion, and syntax highlighting _for each language_
+
+- **Variable sharing** between languages that ensures a continuous workflow within the notebook
+
+    *Variable sharing not available for HTML and Mermaid
+
+- **Variable Explorer** to check state of variables and which subkernel variables exist in
+
+- **Full portability between Jupyter/JupyterLab**
+
+To learn about all the features, visit the VS Code Marketplace [Polyglot Notebooks Extension](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.dotnet-interactive-vscode).
+
+## Prerequisites
+
+To use Polyglot Notebooks in VS Code you will need:
+
+- [Polyglot Notebooks Extension](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.dotnet-interactive-vscode)
+- [.NET 7 SDK](https://dotnet.microsoft.com/en-us/download/dotnet/7.0)
+
+### Why do I need the .NET SDK?
+
+Polyglot Notebooks is powered by .NET Interactive which is a cutting edge and innovative engine that can run multiple languages and share variables between them. In Polyglot Notebooks, it behaves as the notebook kernel and is the reason the .NET 7 SDK is required at this time.
+
+## Getting Started
+
+1. To create your first polyglot notebook, open the command palette (_Ctrl+Shift+P on Windows, Cmd+Shift+P on iOS_) and select `Polyglot Notebooks: Create new blank
```

Comment:
nit nit nit: Would this be "Polyglot Notebook" (with the capital P and N) to symbolize the brand like other places in this markdown?